packages/riipen-ui/src/components/Menu.jsx

```diff
@@ -92,6 +92,8 @@ class Menu extends React.Component {
     closeOnClick: true,
     isOpen: true,
     keepOnScreen: false,
+    onChange: () => {},
+    onClose: () => {},
```

Comment:
Will there ever be a case where someone will want onClose to be undefined when it gets passed into Popover?  I'm kind of wary with empty functions, I usually like making them required as a prop or just undefined, but that's just me 😛

Reply:
hmm, I can see your point. I was just trying to clean up the `if (onClose) onClose()` repeats that were appearing everywhere. Ill revert it for now as there is no reason other than that I was going to change it